src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -844,6 +867,54 @@ def _get_kfp_ui_daemon_port(self) -> int:
             port = networking_utils.find_available_port()
         return port
 
+    def _get_session_cookie(self, username: str, password: str) -> str:
+        session = requests.Session()
+        if (
+            self.config.kubeflow_hostname is None
+            or "/pipeline" not in self.config.kubeflow_hostname
+        ):
+            raise AssertionError(
+                "You must configure the Kubeflow orchestrator "
+                "with the `kubeflow_hostname` parameter which ends "
+                "with `/pipeline` (e.g. `https://mykubeflow.com/pipeline`). "
+                "Please update the current kubeflow orchestrator with: "
+                f"`zenml orchestrator update {self.name} "
+                "--kubeflow_hostname=<MY_KUBEFLOW_HOST>`"
+            )
+        try:
+            response = session.get(self.config.kubeflow_hostname)
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as errh:
+            raise RuntimeError(
+                f"Error while trying to fetch kubeflow cookie: {errh}"
+            )
+        except requests.exceptions.ConnectionError as errc:
+            raise RuntimeError(
+                f"Error while trying to fetch kubeflow cookie: {errc}"
+            )
+        except requests.exceptions.Timeout as errt:
+            raise RuntimeError(
+                f"Error while trying to fetch kubeflow cookie: {errt}"
+            )
+        except requests.exceptions.RequestException as err:
+            raise RuntimeError(
+                f"Error while trying to fetch kubeflow cookie: {err}"
+            )
```

Comment:
```suggestion
        except (requests.exceptions.HTTPError, requests.exceptions.ConnectionError, requests.exceptions.Timeout,  requests.exceptions.RequestException) as e:
            raise RuntimeError(
                f"Error while trying to fetch kubeflow cookie: {e}"
            )
```

This will need some formatting, but you can catch all of them at once like this